test-files/import-errors.ts

```
@@ -0,0 +1,47 @@
+import * as lodash from 'lodash'
+
+// Circular dependency risk
+
+// Default import with wrong name
+
+// Mixed import styles
+
+// Importing non-existent exports
+
+// Dynamic imports in wrong context
+const dynamicImport = import('./dynamic-module')
```

ESLint

[error] 'dynamicImport' is assigned a value but never used. Allowed unused vars must match /^_$/u.

(@typescript-eslint/no-unused-vars)